jobs/cache_maintenance/src/cache_maintenance/main.py

```diff
@@ -15,15 +15,16 @@
 from libcommon.storage import init_duckdb_index_cache_dir
 
 from cache_maintenance.backfill import backfill_cache
+from cache_maintenance.cache_metrics import collect_cache_metrics
 from cache_maintenance.config import JobConfig
 from cache_maintenance.delete_indexes import delete_indexes
-from cache_maintenance.metrics import collect_metrics
+from cache_maintenance.queue_metrics import collect_queue_metrics
 
 
 def run_job() -> None:
     job_config = JobConfig.from_env()
     action = job_config.action
-    supported_actions = ["backfill", "collect-metrics", "delete-indexes", "skip"]
+    supported_actions = ["backfill", "collect-cache-metrics", "collect-metrics", "delete-indexes", "skip"]
```

Comment:
maybe

```suggestion
    supported_actions = ["backfill", "collect-cache-metrics", "collect-queue-metrics", "delete-indexes", "skip"]
```

Reply:
I was planning to change all related names in the other PR to avoid modified files but I can do it here as well.

Reply:
yes, as you want. I proposed it because you renamed metrics.py to queue_metrics.py